modules/tensor_mechanics/examples/coal_mining/tests

```diff
@@ -0,0 +1,14 @@
+[Tests]
+  [./cosserat_elastic]
+    type = 'CSVDiff'
+    input = 'cosserat_elastic.i'
+    csvdiff = 'cosserat_elastic.csv'
+    heavy = true
+  [../]
+  [./cosserat_wp_only]
+    type = RunApp
+    input = 'cosserat_wp_only.i'
+    csvdiff = 'cosserat_elastic.csv'
+    heavy = true
+  [../]
```

Comment:
OK, i can do the syntax test only.  But i do have some CSV data that i can compare with, so do you think two tests: one syntax and the other "heavy" is a good idea?  The "heavy" test isn't incredibly lengthy, but it does take over 2 seconds.